### PR TITLE
Add model deployer to CLI docs

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
Add Model deployer to CLI Docs.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/features/integrations) table.
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: src/zenml/cli/__init__.py [hunk 1]
```diff
@@ -736,6 +736,49 @@
 ```bash
 zenml role assignment list
 ```
+
+Interacting with Model Deployers
+-----------------------------------------
+
+Model deployers are stack components responsible for online model serving.
+They are responsible for deploying models to a remote server, model deployers
```
Comment (strickvl) [inline src/zenml/cli/__init__.py:744]:
```suggestion
They are responsible for deploying models to a remote server. Model deployers
```

File: src/zenml/cli/__init__.py [hunk 2]
```diff
@@ -736,6 +736,49 @@
...
+```
+
+If you wish to delete a particular model deployer, pass the name of the
+model deployers into the CLI with the following command:
+
+```bash
+zenml model-deployer delete MODEL_DEPLOYER_NAME
+```
+
+If you wish to retrieve logs of a particular model deployer, pass the name of the
```
Comment (strickvl) [inline src/zenml/cli/__init__.py:775]:
```suggestion
If you wish to retrieve logs corresponding to a particular model deployer, pass the name of the
```

File: src/zenml/cli/__init__.py [hunk 3]
```diff
@@ -736,6 +736,49 @@
...
+model deployers into the CLI with the following command:
```
Comment (strickvl) [inline src/zenml/cli/__init__.py:776]:
```suggestion
model deployer into the CLI with the following command:
```

**Review by strickvl — CHANGES_REQUESTED** [review]

File: src/zenml/cli/__init__.py [hunk 4]
```diff
@@ -741,7 +741,7 @@
 -----------------------------------------
 
 Model deployers are stack components responsible for online model serving.
-They are responsible for deploying models to a remote server, model deployers
+They are responsible for deploying models to a remote server. model deployers
```
Comment (strickvl) [inline src/zenml/cli/__init__.py:744]:
```suggestion
They are responsible for deploying models to a remote server. Model deployers
```

Sorry! I forgot to make this one a capital M!

**Review by strickvl — CHANGES_REQUESTED** [review]

**Review by AlexejPenner — APPROVED** [review]

**Pull request merged**